src/ProjectAssets/index.tsx

```diff
@@ -1,7 +1,75 @@
+import { useRef } from "react";
+import { useParams } from "react-router-dom";
+
+import { useCreateAssetsMutation, useProjectQuery } from "../@hooks";
+import { ValidationError } from "../@utils/createError";
+import { AssetsList, AssetsListSkeleton } from "./AssetsList";
+import { AsyncBoundary } from "../Error/AsyncBoundary";
+import * as css from "./index.css";
+
+const MB = 1048576;
```

Comment:
```suggestion
const MB = 1024 * 1024;
```

1mb는 1024kb이니까 이렇게 나타내면 더 이해하기 쉬울 것 같습니다~